backend/collab_service/testserver.js

```diff
@@ -0,0 +1,40 @@
+const io = require('socket.io-client');
+
+// Connect to the Socket.IO server
+const socket = io('http://localhost:5001', {
```

Comment:
socket is at 5004

Reply:
`testserver.js` removed, no longer needed